resources/js/components/pm-blocks/CreatePmBlockModal.vue

```diff
@@ -103,7 +123,15 @@ export default {
       },
       name(newValue, oldValue) {
         this.validateName(newValue, oldValue);
-      }
+      },
+      iconAndFile: {
+        handler: function(value) {
+          console.log('ICON AND FILE WATCH CORE', value);
```

Comment:
@mcraeteisha  can you remove this console log please

Reply:
Done!